tests/fuzz/fuzz-wire-closing_complete.c

```diff
@@ -0,0 +1,50 @@
+#include "config.h"
+#include <assert.h>
+#include <ccan/mem/mem.h>
+#include <tests/fuzz/libfuzz.h>
+#include <tests/fuzz/wire.h>
+#include <wire/peer_wire.h>
+
+struct closing_complete {
+	struct channel_id channel_id;
+	u32 locktime;
+	struct amount_sat fee_satoshis;
+	u8 *closer_scriptpubkey, *closee_scriptpubkey;
+	struct tlv_closing_tlvs *tlvs;
+};
+
+static void *encode(const tal_t *ctx, const struct closing_complete *s)
+{
+	return towire_closing_complete(ctx, &s->channel_id, s->closer_scriptpubkey,
+					s->closee_scriptpubkey, s->fee_satoshis, s->locktime, s->tlvs);
+}
+
+static struct closing_complete *decode(const tal_t *ctx, const void *p)
+{
+	struct closing_complete *s = tal(ctx, struct closing_complete);
+
+	if (fromwire_closing_complete(s, p, &s->channel_id, &s->closer_scriptpubkey,
+					&s->closee_scriptpubkey, &s->fee_satoshis, &s->locktime, &s->tlvs))
+		return s;
+	return tal_free(s);
+}
+
+static bool equal(const struct closing_complete *x,
+		  const struct closing_complete *y)
+{
+	assert(!memcmp(&x->channel_id, &y->channel_id, sizeof(struct channel_id)));
+	assert(x->locktime == y->locktime);
+	assert(!memcmp(&x->fee_satoshis, &y->fee_satoshis, sizeof(struct amount_sat)));
```

Comment:
Nit:  for better readability can we use `assert(memcmp(...) == 0)`?  I've seen bugs arise because people thought `!memcmp(...)` meant the buffers differed.

```suggestion
	assert(memcmp(&x->channel_id, &y->channel_id, sizeof(struct channel_id)) == 0);
	assert(x->locktime == y->locktime);
	assert(memcmp(&x->fee_satoshis, &y->fee_satoshis, sizeof(struct amount_sat)) == 0);
```